.github/workflows/test.yml

```diff
@@ -84,6 +85,15 @@ jobs:
         ports:
           - 3306
 
+      mssql:
+        image: mcr.microsoft.com/mssql/server:2019-latest
+        env:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: "Prisma1-prisma"
```

Comment:
Nitpick: why is it called Prisma 1 😅 ? Feels weird to me 🤔

Reply:
Lol, it needs a number in its password, and has to be a certain length for "security". Can change it to `Pr1sma-prisma` if you like that better :P

Reply:
Pr1sm4-Pr1sm4

Reply:
Ok! Sounds good, it's nice to avoid any reference to Prisma 1 I think 😄